lib/web/ui/usercontext.go

```diff
@@ -210,6 +235,7 @@ func NewUserContext(user types.User, userRoles services.RoleSet, features proto.
 		Tokens:          tokenAccess,
 		Nodes:           nodeAccess,
 		Billing:         billingAccess,
+		AWSRoleARNs:     roleARNs,
```

Comment:
@r0mant  UI expects that the roles will be part of the apps object. Currently it does not read them from the UserContext.  If it has to be done this way, it needs to be addressed on the frontend.

@rudream  ^^^

Reply:
@rudream could you also make AWS icon to have always "yellow"  (amazonish) color? :)

Reply:
Will do, and I'll wait for @r0mant's decision regarding the ARN's being in UserContext

Reply:
Yes, role ARNs are returned as a part of the apps as we discussed. I think I added them here originally since that's where all other ACL information is, but since they UI isn't using them from here, I've just removed them now.